CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 ## Unreleased
 
 ### Added
--
+- **cf-core:** [PATCH] Add `u-hidden` utility class.
```

Comment:
This is new functionality and should be `[MINOR]`

Reply:
Updated!